docs/tasks/configuring-a-platform.md

```diff
@@ -163,16 +176,20 @@ healthCheck:
   enabled: false
 ```
 
-### Cron trigger creation mode (`cronTriggerCreationMode`)
-
-A function can run cron triggers as k8s CronJobs or creating the cron logic inside the processor.
-
-For more information - [Cron Trigger](/docs/reference/triggers/cron.md)
+<a id="cronTriggerCreationMode"></a>
+### Cron-trigger creation mode (`cronTriggerCreationMode`)
 
-- `cronTriggerCreationMode`: Which way to implement cron triggers. ["kube", "processor" (default)]
+The `cronTriggerCreationMode` configuration field determines how to run cron triggers:
 
-For example, the following will configure the system to implement cron triggers as k8s CronJobs:
+- `"kube"` - run Cron triggers as Kubernetes CronJobs.
+    <br/>
+    This value is applicable only to Kubernetes platforms and must be set on such platforms.
+- `"processor"` (default)] - run Cron triggers within the processor.
```

Comment:
move first, as it is the default.

Reply:
DONE as part of a full rewrite following our offline discussion in which it was clarified that the default `"processor"` mode is supported also on K8s — see commit 789b8a3.